src/tcgwars/logic/impl/gen3/FireRedLeafGreen.groovy

```diff
@@ -1583,7 +1583,7 @@ public enum FireRedLeafGreen implements LogicCardInfo {
             onAttack {
               damage 10
               afterDamage {
-                apply PARALYZED
+                flipThenApplySC PARALYZED
```

Comment:
`flipThenApplySC` already includes an `afterDamage` clause, so it's best to rewrite this. 
```suggestion
                flip { apply PARALYZED }
```